src/trio/testing/_raises_group.py

```diff
@@ -35,12 +34,16 @@
     from typing import TypeVar
 
     MatchE = TypeVar("MatchE", bound=BaseException, covariant=True)
```

Comment:
Can add a comment that `MatchE` messiness is because `default` was added in py313, and we don't want a runtime dependency on `typing_extensions`